extensions/Worker.Extensions.Http.AspNetCore/src/FunctionsMiddleware/FunctionsHttpProxyingMiddleware.cs

```diff
@@ -44,6 +45,9 @@ public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next
         private static void AddHttpContextToFunctionContext(FunctionContext funcContext, HttpContext httpContext)
         {
             funcContext.Items.Add(Constants.HttpContextKey, httpContext);
+
+            // add asp net version of httprequestdata feature
+            funcContext.Features.Set<IHttpRequestDataFeature>(new AspNetExtensionHttpRequestDataFeature());
```

Comment:
Let's reuse the same instance here so we can avoid an new one with every request (use a pattern similar to what you've done with the default implementation).